docs/source/file_read/file_read.rst

```diff
@@ -0,0 +1,30 @@
+Reading NWB Files
+==================
+
```

Comment:
I think it would be useful to also point users interested in writing NWB data in the right direction. Maybe adding a short introduction statement describing what this section about could do the trick. Maybe something along the below suggestion could work. Note, ```:ref:`user_guide` ``` should point to the conversion tutorial in the docs here, but I think that section may need an anchor.
 
```suggestion
This section provides an introduction on how to read NWB files in Python using PyNWB and Matlab using MatNWB. If you are interested in converting your data to NWB, then please see the :ref:`user_guide` as well as for more in-depth tutorials visit the :pynwb-docs:`PyNWB  <tutorials>` and :matnwb-docs:`MatNWB <#tutorials>` tutorials and documentation.
```